buildSrc/src/main/kotlin/buildsrc/config/Deps.kt

```diff
@@ -16,30 +16,29 @@ object Deps {
         const val coroutines = "1.10.1"
         const val slfj = "2.0.17"
         const val logback = "1.5.18"
-        const val junitJupiter = "5.12.2"
+        const val junitJupiter = "5.14.1"
         const val junit4 = "4.13.2"
 
-        const val byteBuddy = "1.15.11"
-        const val objenesis = "3.3"
-        const val dexmaker = "2.28.3"
-        const val androidxEspresso = "3.5.1"
-        const val androidxOrchestrator = "1.4.2"
-        const val androidxTestRules = "1.5.0"
-        const val androidxTestRunner = "1.5.2"
-        const val androidxTestExtJunit = "1.1.5"
-        const val androidxTestOrchestrator = "1.4.2"
+        const val byteBuddy = "1.18.2"
```

Comment:
This was intentionally downgraded in #1413, but it should be ok to re-upgrade now as discussed in #1434